<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>358</width>
    <height>316</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Введите x:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Результат:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QPushButton" name="pushButton_3">
     <property name="text">
      <string>Вычислить</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
